File: Codewars/8 kyu/Even or Odd/java/src/Main.java
public class Main {
    public static String evenOrOdd(int number) {
        if (number % 2 == 0) {
            return "Even";
        } else {
            return "Odd";
        }
    }

    public static void main(String[] args) {
        // Menguji fungsi
        System.out.println(evenOrOdd(1));
        System.out.println(evenOrOdd(2));
    }
}

File: Codewars/8 kyu/Even or Odd/kotlin/src/main/kotlin/Main.kt
fun evenOrOdd(number: Int): String {
    return if (number % 2 == 0) {
        "Even"
    } else {
        "Odd"
    }
}

fun main(args: Array<String>) {
    // Menguji fungsi
    println(evenOrOdd(1))
    println(evenOrOdd(2))
}